src/commands/deploy.ts

```diff
@@ -311,7 +315,7 @@ export async function deployClasses(configuration: YamlProjectConfiguration, clo
     name: c.className,
     cloudUrl: c.functionUrl
   })));
-  await writeSdkToDisk(sdkResponse, configuration.sdk.language, configuration.sdk.path)
+  await writeSdkToDisk(sdkResponse, configuration.sdk!.language, configuration.sdk!.path)
```

Comment:
I am not a fan of non-null assertion with `!`. I also think they are triggering a warning in the linter.

Is there a way to avoid `!`?

Reply:
checked for `null/undefined` before, so we cand remove `!`